decidim-consultations/app/views/decidim/consultations/consultations/_question.html.erb

```diff
@@ -3,7 +3,7 @@
     <div class="columns mediumlarge-8 large-9 card--process__column">
       <div class="card__content">
         <%= link_to decidim_consultations.question_path(question), class: "card__link" do %>
-          <h2 class="heading5"><%= decidim_sanitize translated_attribute question.title %></h2>
+          <h2 class="heading5"><%= decidim_sanitize translated_attribute(question.title), strip_tags: true %></h2>
```

Comment:
I don't know the original reason why these are WYSIWYG editors in the first place but I'm just thinking do we want to strip all tags here?

Or should we go ahead and fix the problem at the admin side and change this field to a normal text input?

But that would also require introducing a new migration to migrate all the old HTML data into the plain text data which is problematic for backports. Otherwise the plain text field would show HTML tags in it if the field type is changed.